pkg/storage/storage.go

```diff
@@ -455,27 +455,28 @@ func (s *storageModule) ListFilesystems() ([]pkg.Filesystem, error) {
 // Path return the path of the mountpoint of the named filesystem
 // if no volume with name exists, an empty path and an error is returned
 func (s *storageModule) Path(name string) (pkg.Filesystem, error) {
-	for idx := range s.pools {
```

Comment:
Is there a particular reason that these loops are changed from index based access to value copy access? Since there seems to be a lot of changes here which don't add anything

Reply:
No reason, was looking at the code and I don't think using indexes in for loops is very readable. That's why I changed this

Reply:
Thats actually highly opinionated. I for one think that this is perfectly readable. I'm not really a fan of changing code because someone thinks it looks prettier in a certain way, else you end up with a bunch of pull requests all changing code back and forth. As long as it passes the linters it should be fine.

Also, while it is not the case here, this change is not purely cosmetic. There is actually a subtle difference in looping based on indexes, or by assigning a value.

Reply:
As I aggree with you, I also think consistency is a good thing. The most of provisiond's for loops are not using indexes since the disk shutdown feature was introduced. Ideally we want code to be consistent. But I will revert if you insist.